components/sections/PrefSection.vue

```diff
@@ -59,6 +62,17 @@ useSelectAll(fontSizeInput)
         v-model.number="options.rootFontSize"
       />
     </div>
+    <div class="tp-row tp-row-justify tp-pref-field">
+      <label for="scale-multiplier">Scale multiplier</label>
```

Comment:
```suggestion
      <label for="scale">Scale</label>
```

I think we don't need "multiplier".